examples/Hermitian_Phasing.py

```diff
@@ -3,18 +3,18 @@
 ============================================
 
 Take fake data with a relatively symmetric echo 
-(:math:`T_2^*=1/50\pi`, echo time of 10 ms),
+(:math:`T_2^*=1/50 pi`, echo time of 10 ms),
```

Comment:
Bad change

Reply:
You want to add r to the triple quotes

Reply:
see [here](https://jmfrancklab.slack.com/archives/C016KQGJY7R/p1721771227229949)